### PR TITLE
initial solaris solaris suppport

### PR DESCRIPTION
**Pull request by gdams**
test build passing here https://ci.adoptopenjdk.net/job/jdk8u-solaris-sparcv9-hotspot-test/

### CODE REVIEW

File: sbin/common/config_init.sh [hunk 1]
```diff
@@ -309,7 +309,12 @@ function configDefaults() {
   BUILD_CONFIG[FREETYPE_FONT_VERSION]="2.9.1"
   BUILD_CONFIG[FREETYPE_FONT_BUILD_TYPE_PARAM]=""
 
-  BUILD_CONFIG[MAKE_COMMAND_NAME]="make"
+  if [ "${BUILD_CONFIG[OS_KERNEL_NAME]}" == "aix" ] || [ "${BUILD_CONFIG[OS_KERNEL_NAME]}" == "SunOS" ]; then
```
Comment (karianna) [inline sbin/common/config_init.sh]: Are we not covering this AIX case elsewhere?  How is it currently working?

File: sbin/prepareWorkspace.sh [hunk 2]
```diff
@@ -47,7 +47,7 @@ checkoutAndCloneOpenJDKGitRepo()
     set +e
     git --git-dir "${BUILD_CONFIG[OPENJDK_SOURCE_DIR]}/.git" remote -v
     echo "${BUILD_CONFIG[OPENJDK_CORE_VERSION]}"
-    git --git-dir "${BUILD_CONFIG[OPENJDK_SOURCE_DIR]}/.git" remote -v | grep "origin.*fetch" | grep "${BUILD_CONFIG[OPENJDK_CORE_VERSION]}" | grep --quiet "${BUILD_CONFIG[REPOSITORY]}"
+    git --git-dir "${BUILD_CONFIG[OPENJDK_SOURCE_DIR]}/.git" remote -v | grep "origin.*fetch" | grep "${BUILD_CONFIG[OPENJDK_CORE_VERSION]}" | grep "${BUILD_CONFIG[REPOSITORY]}"
```
Comment (karianna) [inline sbin/prepareWorkspace.sh]: I think we still want quiet....

Comment (gdams) [reply sbin/prepareWorkspace.sh]: quiet isn't a flag on solaris grep and this only logs a single line matching the repo so IMO not much noise

File: sbin/common/config_init.sh (hunk 1 above)
Comment (gdams) [reply sbin/common/config_init.sh]: AIX was hacked on the machines whereas I've now made it universal

**Pull request merged**